### ISSUE TITLE
Mobile Responsiveness: Top banner containing the Usbong apps has disappeared

### ISSUE THREAD

**Issue by masarapmabuhay** [usbong/UsbongStore#100]
Hi Zent,
I hope you are doing well.

I observed that when the mobile responsiveness setting is turned on, 
the top banner containing the Usbong apps has disappeared.

May I kindly ask if this behavior is really included in your design?

Thank you.

Peace,
Mike

**Comment by zent-lim** [comment]
Hi Mike,

Yes.

(1) The app bar wasn't part of the scope for mobile responsiveness:

      Ref:
https://drive.google.com/open?id=1egJxDvrBUJfiNzpHHUEzUZ0rslqyXYW0

      Section: "(1) The header containg app icons will not be touched."

(2) It was hidden on xs for better user experience.

IMPORTANT:

(1) In mobile responsiveness, user expects less information for a less
cluttered screen. It's an accepted user experience principle to show less
important data on smaller screen.

(2) I removed the app bar on xs because:

      (a) it's secondary information

      (b) The primary information is to display beverages.

(3) If you add the app bar on XS it's a minus on user experience because:

     (a) on opening the page, user is expecting to see beverages

     (b) instead he will see several rows of app icons on xs instead
(unlike in bigger screens where it only occupies one row, in xs it will
occupy around 10 rows.)

     This will likely increase bounce rate.

Morning!

**Comment by zent-lim** [comment]
Hi Mike,

Someone with a design background might be able to help you better.

Here's one of several ways to do it (although my way is likely crude compared to folks with real design background).

Implement a 1 image carousel for the apps before the footer.

(1) what is a carousel?

![screenshot-2018-7-7 usbong store](https://user-images.githubusercontent.com/8021327/42407389-20416fc0-81ee-11e8-9864-90abef413e33.png)

(2) how do I implement a carousel using bootstrap?

    https://getbootstrap.com/docs/3.3/javascript/#carousel

(3) why only a one image carousel?

     Cause you're on xs screen. Most likely you can only fit one image.

(4) why before the footer?

      Basically, after the primary data (products).

(5) what do I need to get started?

     Most likely, you need bigger images for the app icons. 

Afternoon!
